elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal--Padding--mobile: 0 0 0 calc(#{pfe-var(container-padding)} * 2);
```

Comment:
Instead of creating a separate mobile variable, could we update the original padding variable inside a media query in the host selector?  So...
```
:host {
     --pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
     @media screen and (max-height: $pfe-modal--breakpoint--medium) {
          --pfe-modal--Padding: 0 0 0 calc(#{pfe-var(container-padding)} * 2);
     }
}
```

Reply:
Done. Much cleaner now. Thanks.